docker/Dockerfile.1-13-gpu

```diff
@@ -0,0 +1,65 @@
+ARG TFS_VERSION
+
+FROM tensorflow/serving:${TFS_VERSION}-gpu as tfs
+FROM nvidia/cuda:10.0-base-ubuntu16.04
+LABEL com.amazonaws.sagemaker.capabilities.accept-bind-to-port=true
+
+COPY --from=tfs /usr/bin/tensorflow_model_server /usr/bin/tensorflow_model_server
+
+# https://github.com/tensorflow/serving/blob/1.13.0/tensorflow_serving/tools/docker/Dockerfile.gpu
+ENV CUDNN_VERSION=7.4.1.5
+ENV TF_TENSORRT_VERSION=5.0.2
+
```

Comment:
I saw NCCL is installed in current dockerfile https://github.com/aws/sagemaker-tensorflow-serving-container/blob/master/docker/Dockerfile.gpu, is there a reason removing it?

Reply:
NCCL is part of TF core now. We do not need to install it anymore - https://www.reddit.com/r/tensorflow/comments/am7lnc/where_is_nccl_in_tf_113/